versions/g-/gmp.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "79b2b7cf5ffa0ceb6b769efbfe90ebc6fb0db430",
+      "version-semvar": "6.2.1",
```

```suggestion
      "version-semver": "6.2.1",
```
Please run `vcpkg x-add-version --overwrite-version gmp` to update, not do this change manually.

---

Which version of vcpkg has the `x-add-version` option?  Here what I got.

```
❯ ./vcpkg version
Vcpkg package management program version 2021-07-26-unknownhash

See LICENSE.txt for license information.
❯ ./vcpkg x-add-verion --overwrite-version gmp
invalid command: x-add-verion
Commands:
  vcpkg search [pat]              Search for packages available to be built
  vcpkg install <pkg>...          Install a package
  vcpkg remove <pkg>...           Uninstall a package
  vcpkg remove --outdated         Uninstall all out-of-date packages
  vcpkg list                      List installed packages
  vcpkg update                    Display list of packages for updating
  vcpkg upgrade                   Rebuild all outdated packages
  vcpkg x-history <pkg>           (Experimental) Shows the history of CONTROL versions of a package
  vcpkg hash <file> [alg]         Hash a file by specific algorithm, default SHA512
  vcpkg help topics               Display the list of help topics
  vcpkg help <topic>              Display help for a specific topic

  vcpkg integrate install         Make installed packages available user-wide
  vcpkg integrate remove          Remove user-wide integration
  vcpkg integrate bash            Enable bash tab-completion
  vcpkg integrate x-fish          Enable fish tab-completion

  vcpkg export <pkg>... [opt]...  Exports a package
  vcpkg edit <pkg>                Open up a port for editing (uses $EDITOR, default 'code')
  vcpkg create <pkg> <url> [archivename]
                                  Create a new package
  vcpkg x-init-registry <path>    Initializes a registry in the directory <path>
  vcpkg owns <pat>                Search for files in installed packages
  vcpkg depend-info <pkg>...      Display a list of dependencies for packages
  vcpkg env                       Creates a clean shell environment for development or compiling
  vcpkg version                   Display version information
  vcpkg contact                   Display contact information to send feedback

Options:
  --triplet=<t>                   Specify the target architecture triplet. See 'vcpkg help triplet'
                                  (default: $VCPKG_DEFAULT_TRIPLET)
  --host-triplet=<t>              Specify the host architecture triplet. See 'vcpkg help triplet'
                                  (default: $VCPKG_DEFAULT_HOST_TRIPLET)
  --overlay-ports=<path>          Specify directories to be used when searching for ports
                                  (also: $VCPKG_OVERLAY_PORTS)
  --overlay-triplets=<path>       Specify directories containing triplets files
                                  (also: $VCPKG_OVERLAY_TRIPLETS)
  --binarysource=<path>           Add sources for binary caching. See 'vcpkg help binarycaching'
  --downloads-root=<path>         Specify the downloads root directory
                                  (default: $VCPKG_DOWNLOADS)
  --vcpkg-root=<path>             Specify the vcpkg root directory
                                  (default: $VCPKG_ROOT)
  --x-buildtrees-root=<path>      (Experimental) Specify the buildtrees root directory
  --x-install-root=<path>         (Experimental) Specify the install root directory
  --x-packages-root=<path>        (Experimental) Specify the packages root directory
  --x-scripts-root=<path>         (Experimental) Specify the scripts root directory
  --x-builtin-ports-root=<path>   (Experimental) Specify the packages root directory
  --x-builtin-registry-versions-dir=<path>
                                  (Experimental) Specify the versions root directory
  --x-json                        (Experimental) Request JSON output

  @response_file                  Specify a response file to provide additional parameters

For more help (including examples) see the accompanying README.md and docs folder.
```

---

`x-add-version` was supported by PR https://github.com/microsoft/vcpkg/pull/15664. This changes were merged to vcpkg 2021.04.30. 

I update `version-semvar` as `version-semver` manually (since `version-semvar` cannot be recognized) and then run `vcpkg x-add-version --overwrite-version gmp` successfully.


```
PS F:\18610\vcpkg> .\vcpkg.exe x-add-version --overwrite-version gmp
Error: Unable to parse versions file F:\18610\vcpkg\versions\g-\gmp.json.
Error: failed to parse versions file for `gmp`:
$.versions[0] (a version database entry): unexpected field 'version-semvar', did you mean 'version-semver'?
$.versions[0] (a version database entry): unexpected "port_version" without a versioning field
PS F:\18610\vcpkg> .\vcpkg.exe x-add-version --overwrite-version gmp
```
On your branch, vcpkg version is `2021-05-05`.
```
PS F:\18610\vcpkg> .\vcpkg.exe version
Vcpkg package management program version 2021-05-05-b110959e13f562c2a17d7bd9e352e2dd8d6dc603

See LICENSE.txt for license information.
PS F:\18610\vcpkg> .\vcpkg.exe x-add-version --overwrite-version gmp
PS F:\18610\vcpkg>
```